Project.toml

```diff
@@ -4,17 +4,17 @@ authors = ["Pietro Vertechi"]
 version = "0.3.2"
 
 [deps]
-BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
```

Comment:
This seems like a change from #53, so it shouldn't be in this PR. Same for many of the changes below.

Reply:
Indeed that is the case. Will close this PR.